libs/SalesforceSDKCore/SalesforceSDKCore/Classes/Extensions/RestClient.swift

```diff
@@ -115,6 +116,7 @@ extension RestClient {
     /// Execute a prebuilt request.
     /// - Parameter request: `RestRequest` object.
     /// - Parameter completionBlock: `Result` block that handles the server's response.
+    @available(*, deprecated, message: "Use the async/await version of `send(request:)` instead.")
```

Comment:
I think we should add something to the deprecation messages about it being removed in 14.0. We have [this](https://github.com/forcedotcom/SalesforceMobileSDK-iOS/blob/40bcc28b8b96b5764870e306db9c4143b9a7c007/libs/SalesforceSDKCore/SalesforceSDKCore/Classes/Common/SalesforceSDKConstants.h#L103) for deprecations in Objective-C, I'm not sure if there's a way to do something similar to have a standard message in Swift

Reply:
Got it makes sense.  I'll add that.

Reply:
one small thing, can you use 13.0 and 14.0 (vs 13 and 14) in the deprecation messages?

Reply:
Got it.  I'll update that.